### DISCUSSION TITLE
【求助】到底该选哪个好...

### DISCUSSION THREAD

**Discussion by DipleMoss** [share121/inter-knot#1184] [其他]
我手上的丁尼不多了，我在想我到底该买什么口味的三明治，我喜欢的口味分别是「烟熏鸡肉三明治」「金枪鱼蟹肉三明治」「鸡蛋火腿三明治」「咖喱牛肉三明治」，有没有人能帮我决策一下((つ≧▽≦)つ

**Comment by yuki15532-cpu** [comment]
要不折中一下，吃三明治的面包片

**Reply by DipleMoss** [reply]
我认为这不是一个很好的决定...不过还是谢谢你啦~
